app/server/api/search_api.py

```diff
@@ -85,22 +163,47 @@ def get(self):
 
             # Then use those results to join aginst TransferAccount or CreditTransfer
             if search_type == 'transfer_accounts':
+            # TransferAccount Search Logic
                 final_query = db.session.query(TransferAccount)\
                     .join(user_search_result, user_search_result.c.default_transfer_account_id == TransferAccount.id)\
-                    .order_by(db.text('rank DESC'))\
+                    .join(User, user_search_result.c.default_transfer_account_id == User.default_transfer_account_id)\
+                    .execution_options(show_all=True)\
```

Comment:
I don't think we want `.execution_options(show_all=True)` anywhere on this API at this time... otherwise, all organisation data will be returned.

Reply:
Ditto about merge conflicts, fixed